expan/core/util.py

```diff
@@ -180,3 +109,9 @@ def generate_random_data_n_variants(n_variants=3):
 
     return test_data_frame, metadata
 
+
+def get_kpi_by_name_and_variant(data, name, variant):
```

Comment:
this functionality can be part of `Variant` and `FeatureFilter` classes

Reply:
do you mean make it as a class method for both Variant and FeatureFilter? wouldn't it be a code duplicate?

Reply:
here means Variant and KPI?